pkg/vorteil/launch.go

```diff
@@ -105,10 +105,13 @@ func (p *program) waitForApp(cmd *exec.Cmd) {
 	err := cmd.Wait()
 	if err != nil {
 		logDebug("error while waiting: %s", err.Error())
-	} else {
-		logDebug("process %d finished with %s", cmd.Process.Pid, cmd.ProcessState.String())
```

Comment:
We should probably check if .Process is not nil. There are edge cases where that might be nil